### ISSUE TITLE
Input group right element button dancing

### ISSUE THREAD

**Issue by Reaverart** [palantir/blueprint#237]
### Bug Report

- __Package version(s)__: (1.1.0)
- __Browser and OS versions__: (chrome 54, osX 10.11.6)

#### Describe the issue:

If input group lays inside of control group a rightElement button moving on click. 
It happens because of this rule https://github.com/palantir/blueprint/blob/master/packages/core/src/components/forms/_control-group.scss#L130
So 'pt-input-action' becomes 0 width and button moves right, because it aligns to left if no side specified
```
<div class="pt-control-group">
  <div class="pt-input-group">
    <span class="pt-icon pt-icon-search"></span>
    <input type="text" placeholder="Search..." class="pt-input" style="padding-right: 30px;">
    <span class="pt-input-action">
      <button class="pt-button pt-icon-small-cross pt-intent-warning pt-minimal" type="button"></button>
    </span>
  </div>
</div>
```
#### Actual behavior:

button moves right
![repro](https://cloud.githubusercontent.com/assets/443450/20614990/ffe0c54e-b289-11e6-88c6-e62b71d4e9e7.gif)
_EDIT: GIF added by @cmslewis_

#### Expected behavior:

button should stay on same position

**Comment by cmslewis** [comment]
Thanks for filing this bug, @Reaverart. Really appreciate the code sample too - definitely helps us move faster to repro the issue. Seems quick enough that we should hopefully get a fix in shortly.

**Comment by cmslewis** [comment]
Fixed by #239
